src/main/java/org/maxgamer/rs/util/log/Log.java

```diff
@@ -1,117 +1,43 @@
 package org.maxgamer.rs.util.log;
 
-import org.maxgamer.rs.util.log.Logger.LogLevel;
-
-import java.io.File;
-import java.io.FileOutputStream;
-import java.io.IOException;
-import java.io.PrintStream;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * @author netherfoam
  */
 public class Log {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1118)